### ISSUE TITLE
Build: Name libbalistica properly

### ISSUE THREAD

**Issue by steveno** [fusilero/balistica#25]
```
[100%] Built target lib-balistica_test
Install the project...
-- Install configuration: ""
-- Up-to-date: /usr/local/share/applications/balistica.desktop
-- Up-to-date: /usr/local/share/appdata/balistica.appdata.xml
-- Up-to-date: /usr/local/share/icons/hicolor/16x16/apps/balistica.png
-- Up-to-date: /usr/local/share/icons/hicolor/24x24/apps/balistica.png
-- Up-to-date: /usr/local/share/icons/hicolor/32x32/apps/balistica.png
-- Up-to-date: /usr/local/share/icons/hicolor/48x48/apps/balistica.png
-- Up-to-date: /usr/local/share/icons/hicolor/128x128/apps/balistica.png
-- Up-to-date: /usr/local/share/icons/hicolor/256x256/apps/balistica.png
-- Up-to-date: /usr/local/share/icons/hicolor/512x512/apps/balistica.png
-- Up-to-date: /usr/local/share/icons/hicolor/scalable/apps/balistica.svg
-- Not updating desktop database
-- Up-to-date: /usr/local/share/gnome/help/balistica/C/basic_usage.page
-- Up-to-date: /usr/local/share/gnome/help/balistica/C/bugs.page
-- Up-to-date: /usr/local/share/gnome/help/balistica/C/greenhill.page
-- Up-to-date: /usr/local/share/gnome/help/balistica/C/index.page
-- Up-to-date: /usr/local/share/gnome/help/balistica/C/introduction.page
-- Up-to-date: /usr/local/share/gnome/help/balistica/C/limitations.page
-- Up-to-date: /usr/local/share/gnome/help/balistica/C/miller_twist.page
-- Up-to-date: /usr/local/share/balistica/sql/Version-001.sql
-- Up-to-date: /usr/local/share/balistica/sql/Version-002.sql
-- Up-to-date: /usr/local/share/balistica/sql/fill_bullets.sql
-- Up-to-date: /usr/local/share/balistica/sql/fill_calibers.sql
-- Up-to-date: /usr/local/share/balistica/sql/fill_gauges.sql
-- Installing: /usr/local/lib/liblib-balistica.a
-- Installing: /usr/local/include/lib-balistica.h
-- Installing: /usr/local/bin/balistica
```

**Comment by steveno** [comment]
Blocked while working on https://github.com/steveno/balistica/pull/26

**Comment by steveno** [comment]
Closed by #26
